vector/src/main/res/layout/fragment_matrix_to_room_space_card.xml

```diff
@@ -89,6 +91,13 @@
             tools:text="Public Space"
             tools:visibility="visible" />
 
+        <androidx.constraintlayout.widget.Barrier
+            android:id="@+id/header_barrier"
```

Comment:
According to the naming convention for ids in this file, maybe rename to `matrixToHeaderBarrier`